src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -9,33 +16,34 @@ export const PsvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
       type: FormNodeTypes.CONTROL,
       viewType: FormNodeViewTypes.VEHICLETYPE
     },
     {
       name: 'regnDate',
       label: 'Date of first registration',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.DATE
+      viewType: FormNodeViewTypes.DATE,
+      editType: FormNodeEditTypes.DATE,
+      validators: [{ name: ValidatorNames.Required }],
+      isoDate: false
     },
     {
       name: 'manufactureYear',
       label: 'Year of manufacture',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      editType: FormNodeEditTypes.NUMBER,
+      validators: [{ name: ValidatorNames.MaxLength, args: 4 }, { name: ValidatorNames.Required }]
     },
     {
       name: 'noOfAxles',
       label: 'Number of axles',
```

Comment:
Also wondering whether we should be displaying noOfAxles within vehicle summary along with axle fitted with parking brake? Might be one to ask service design where it belongs
![image](https://user-images.githubusercontent.com/92087051/196949070-f303745b-f8f5-4e67-8ee8-34f00ae381b8.png)

Reply:
After reading your comment above, I think it's worth us removing the axle fields in case we forget to down the line, plus it means that we're matching up with intended service design and saying that vehicle summary section is done without having to revisit it in future

Reply:
(realise that axle fitted with parking brake is on service design, not sure what approach to take for what they have on that service design, I think it's worth a conversation)